apps/public-reference/pages/applications/contact/address.tsx

```diff
@@ -24,12 +24,7 @@ import { PhoneField } from "@bloom-housing/ui-components/src/forms/PhoneField"
 
 export default () => {
   const context = useContext(AppSubmissionContext)
-  const { application, listing } = context
-  const conductor = useMemo(() => new ApplicationConductor(application, listing, context), [
-    application,
-    listing,
-    context,
-  ])
+  const { conductor, application, listing } = context
```

Comment:
Much better!!!

I'd probably write it on a single line, rather than 2, but that's _purely_ stylistic:

```typescript
const { conductor, application, listing } = useContext(AppSubmissionContext)
```

Reply:
@bencpeters All right I checked in simplified syntax in all these cases. Build is failing though, but it looks like that's an issue on master so will investigate separately.